### DISCUSSION TITLE
InvalidArgumentException for test page

### DISCUSSION THREAD

**Discussion by tinusmile** [statamic/cms#5135] [Troubleshooting] [answered]
I am new on Statamic and just run into an error during local install. I am not able to access the frontend of my first site. I get the following error:
«Passed value cannot be an array» at vendor/statamic/stringy/src/Stringy.php:45
I have installed Laravel Valet and latest Statamic.
Your help is much appreciated!

<img width="1225" alt="Cursor_and_🧨_Passed_value_cannot_be_an_array" src="https://user-images.githubusercontent.com/62757/151172159-f57fe349-fa3b-4f7d-9b1d-04e2cc7ffd07.png">
!

**Comment by duncanmcclean** [comment, marked as answer]
Welcome to Statamic! This is actually a known issue (#5126) which will be fixed in the next release.

In the meantime, you can downgrade to Laravel 8.80 (8.81 broke it) and it'll be fixed for now.

**Reply by tinusmile** [reply]
Ok, I did that. Unfortunately there seems to be another problem. monolog requires a lower PHP version mine is 8.1.2:

Terminal:
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Root composer.json requires monolog/monolog 2.0.* -> satisfiable by monolog/monolog[2.0.0, 2.0.1, 2.0.2].
    - monolog/monolog[2.0.0, ..., 2.0.2] require php ^7.2 -> your php version (8.1.2) does not satisfy that requirement.
  Problem 2
    - laravel/framework[v8.80.0, ..., v8.81.0] require monolog/monolog ^2.0 -> satisfiable by monolog/monolog[2.0.0, 2.0.1, 2.0.2].
    - monolog/monolog[2.0.0, ..., 2.0.2] require php ^7.2 -> your php version (8.1.2) does not satisfy that requirement.
    - Root composer.json requires laravel/framework ^8.80 -> satisfiable by laravel/framework[v8.80.0, v8.81.0].
Installation failed, reverting ./composer.json to its original content.

Thanks again!

**Reply by ryanmitchell** [reply]
A new release has just come out that should fix this

**Reply by tinusmile** [reply]
Thanks a lot, it's working as expected now!
